daily-challenges/2024-11/07/ththao27.spec.ts

```
@@ -0,0 +1,70 @@
+function caesarCipher (massages: string[], shift: number): string[] {
+    shift = shift % 26;
+    const result: string[] = [];
+    for (const item of massages) {
+        result.push(encryptionCaesar(item, shift));
+    }
+    return result;
+}
+
+function encryptionCaesar (massage: string, shift: number): string {
+    let result = '';
+    for (let char of massage) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'char' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)